src/NServiceBus.RavenDB/Outbox/RavenDbOutboxStorage.cs

```diff
@@ -37,7 +37,8 @@ protected override void Setup(FeatureConfigurationContext context)
                 .ConfigureProperty(c => c.StoreToInstall, store);
 
             context.Container.ConfigureComponent<OutboxPersister>(DependencyLifecycle.InstancePerCall)
-                .ConfigureProperty(x => x.DocumentStore, store);
+                .ConfigureProperty(x => x.DocumentStore, store)
+                .ConfigureProperty(x => x.EndpointName, context.Settings.EndpointName());
```

Comment:
Lol, was about to make the same comment but caught my self, see https://github.com/Particular/NServiceBus.RavenDB/pull/114/#issuecomment-156378889